pkg/cluster/driver/libvirt/image.go

```diff
@@ -32,6 +33,7 @@ func checkLibvirtError(err error, ec libvirt.ErrorNumber) bool {
 // container image to the target libvirtd instance, locating in the given
 // Pool and assigning in the given name.
 func TransferBaseImage(l *libvirt.Libvirt, imageName string, pool *libvirt.StoragePool, volumeName string, arch string) error {
+	log.Debugf("Downloading bootVolumeContainerImage %s", imageName)
```

Comment:
This debug statement should be inside EnsureBaseQcow2Image (which may or may not download)

Reply:
Ok! Thanks!